test/raft/unit/test_snapshot.c

```diff
@@ -8,6 +8,7 @@
 #include "../../../src/raft.h"
```

Comment:
Curious, why this file is not a part of dqlite-tests?

Reply:
Please ignore this comment. The initial question was whether the `test/raft/unit/test_snapshot.c` shall be a part of dqlite or raft.